tests/e2e/manifest/reset.sh

```
@@ -7,6 +7,17 @@
 # Apply ElastiService
 kubectl apply -f  "$1/target-elastiservice.yaml" -n target
 
+# Wait for ElastiService to be old enough
+CRD_CONTENT=$(kubectl get elastiservices/target-elastiservice -n target -o json)
+CRD_COOLDOWN_PERIOD=$(echo "${CRD_CONTENT}" | jq -r .spec.cooldownPeriod)
+CRD_CREATION_DATE=$(echo "${CRD_CONTENT}" | jq -r .metadata.creationTimestamp)
+CRD_CREATION_SECONDS=$(date -d "${CRD_CREATION_DATE}" +%s)
+SECONDS_NOW=$(date +%s)
+CRD_AGE=$(($SECONDS_NOW - $CRD_CREATION_SECONDS))
```

Shellcheck (0.10.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)